server/index-node/src/schema.graphql

```diff
@@ -19,6 +19,7 @@ type Query {
     indexer: Bytes
   ): Bytes
   subgraphFeatures(subgraphId: String!): SubgraphFeatures!
+  subgraphVersions(subgraphId: String!): [SubgraphVersion!]!
```

Comment:
This is for the Gateway.

Reply:
Nice! We should avoid the words 'subgraph version' here since it already has a meaning in the context of `graph-node` as the mapping of subgraph names to deployments (IPFS hash) How about calling this `apiVersions`?  (That naming change should of course happen throughout the code supporting this GraphQL API)

Reply:
Done